src/index.js

```diff
@@ -34,10 +34,11 @@ ReactDOM.render(
     <Auth0Provider
       domain={config.domain}
       client_id={config.clientId}
-      redirect_uri={window.location.origin+'/dev-sean/'}
+      redirect_uri={window.location.origin}
       onRedirectCallback={onRedirectCallback}
+      returnTo={window.location.origin}
```

Comment:
Does this need to be here?

Reply:
It was from our group conversation yesterday of the code snip I asked for, I can remove it